web/src/backend/SyncerBackend.js

```diff
@@ -54,3 +54,10 @@ export function deleteSyncer(syncer) {
     body: JSON.stringify(newSyncer),
   }).then(res => res.json());
 }
+
+export function runSyncer(owner, name) {
+  return fetch(`${Setting.ServerUrl}/api/run-syncer?id=${owner}/${encodeURIComponent(name)}`, {
+    method: 'GET',
+    credentials: 'include',
+  }).then(res => res.json());
+}
```

Comment:
new line

Reply:
I added the new line but it seems git delete it automatically

![image](https://user-images.githubusercontent.com/33992371/163770520-6ce3e5d3-0ee6-4a13-ae7f-9cdcf62bf7aa.png)